monitoring/views/grouped_schema_error_counts.view.lkml

```diff
@@ -0,0 +1,23 @@
+include: "//looker-hub/monitoring/views/schema_error_counts.view.lkml"
+
+view: grouped_schema_error_counts {
+    extends: [schema_error_counts]
+
+  sql_table_name:`moz-fx-data-shared-prod`.monitoring.schema_error_counts;;
```

Comment:
should we just extend the existing `schema_error_counts` view here? Having separate `schema_error_counts` and `grouped_schema_error_counts` explores might be more confusing since they are _very_ similar.

```suggestion
view: +schema_error_counts {
```

Reply:
ah, this works much better. Thanks!